flow_py_sdk/client/entities.py

```diff
@@ -288,12 +308,22 @@ def from_proto(
         proto: access.TransactionResultResponse,
         id: bytes,
     ) -> "TransactionResultResponse":
+        events = []
+        for i, event_proto in enumerate(proto.events):
+            try:
+                event = Event.from_proto(event_proto)
+                if event is not None:
+                    events.append(event)
+            except Exception as e:
+                logging.error(f"Failed to deserialize event {i}: {e}")
+                continue
+
         return TransactionResultResponse(
             id_=id,
             status=proto.status,
             status_code=proto.status_code,
             error_message=proto.error_message,
-            events=[Event.from_proto(e) for e in proto.events],
+            events=events,
```

Comment:
@janezpodhostnik initially when debugging I considered introducing a max limit on number events processed per transaction as I thought it might be causing the issue... in the end it didn't seem to have much impact on this specific transaction so I opted not to pursue these changes for now.

Reply:
Will resolve this comment for time being and we can look into introducing a restriction on event count if we run into further issues